src/pages/setup/setProfile/SetJobPage.tsx

```
@@ -3,13 +3,46 @@ import { useNavigate } from 'react-router';
+
     const handleSelect = (value: string) => {
         setSelected(value);
     };
+
+    const handleSubmit = async () => {
+        if (isNextDisabled) return;
+
+        const updatedUserInfo = { ...userInfo, category: selected };
+
+        setUserInfo(updatedUserInfo);
+        debugger;
```

Biome (2.1.2)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)